src/compiler/emitter.ts

```diff
@@ -2073,9 +2074,7 @@ namespace ts {
             for (let i = 0; i < lines.length; i++) {
                 const line = indentation ? lines[i].slice(indentation) : lines[i];
                 if (line.length) {
-                    if (i > 0) {
-                        writeLine();
-                    }
+                    writeLine();

```

Comment:
Looks like a merge issue, but this doesn't result in multiple lines. `writeLine()` doesn't actually write a line, it just sets a flag to write a line before the next `write()`, so multiple lines are always reduced to a single line.